inexor/crashreporter/StackWalker_windows.cpp

```diff
@@ -0,0 +1,1025 @@
+///////////////////////////////////////////////////////////////////////////
```

Comment:
yeah .. :/ i didnt want to write the platform stuff myself so this part is actualyl a lib